unpub/lib/src/app.dart

```diff
@@ -27,8 +27,7 @@ class App {
   final String upstream;
   final String googleapisProxy;
   final String overrideUploaderEmail;
-  final Future<void> Function(
-      Map<String, dynamic> pubspec, String uploaderEmail) uploadValidator;
+  final Future<void> Function(Map<String, dynamic> pubspec, String uploaderEmail) uploadValidator;
```

Comment:
@andrelewkatz I think it might be a good idead to not commit this, so we can try to PR in the main repo latter with less noise. What do you think?

Reply:
You're suggesting to undo the formatting?